rds.tf

```diff
@@ -1,3 +1,7 @@
+locals {
+  external_reporting_stg_count = "${var.environment == "prod" ? "1" : "1"}"
```

Comment:
external_reporting_stg_count = "${var.environment == "prod" ? "1" : "0"}"

Reply:
As Ian suggested this still need changing.